arb_os/signedTx.mini

```diff
@@ -27,7 +27,7 @@ import func bytearray_copy(
 import func bytestream_new(ba: ByteArray) -> ByteStream;
 import func bytestream_skipBytes(ba: ByteStream, nbytes: uint) -> option<ByteStream>;
 
-import func rlp_encodeAndHashMessageInfo(tx: SignedTx) -> bytes32;
+import func rlp_encodeAndHashMessageInfoForSignature(tx: SignedTx, eip155v: option<uint>) -> bytes32;
```

Comment:
It might be clearer to make this param `eip155ChainID` or just `chainID`

Reply:
Fixed.